client/source/components/RelatedProducts/RelatedList.jsx

```diff
@@ -22,6 +29,7 @@ const RelatedList = function CreateRelatedList() {
       .catch((error) => { console.error(error); });
   }, [prodID]);
 
+  // HANDLERS
   // const handlePrevious = () => {
   //   const newIndex = index - 1;
   //   setIndex(newIndex < 0 ? length - 1 : newIndex);
```

Comment:
Main should be clean ideally, not in a development state. Consider instead having folks merge your branch into theirs, instead.

**branch >>> branch** instead of
**branch >>>merge>>> main >>>merge>>> their branch**